index.bs

```diff
@@ -144,6 +128,17 @@ partial interface DocumentOrShadowRoot {
 	</dd>
 </dl>
 
-Applying Styles In All Contexts {#styles-in-all-contexts}
-===================
+Every {{DocumentOrShadowRoot}} has adopted stylesheets (of type FrozenArray&lt;CSSStyleSheet>).
```

Comment:
Here is where you should add a normative requirement about how to treat the adopted stylesheets. For example

```
For documents, the user agent must include all style sheets in the {{Document}}'s
[=adopted stylesheets=] inside its <a spec=cssom-1>document CSS style sheets</a> collection,
ordered after all the other style sheets (i.e. those derived from {{Document/styleSheets}}).
```

Also, we need some statement for how this affects shadow roots?

Reply:
I added this but combined it with shadow root also. I don't know where "shadow stylesheets" is defined though..